app/javascript/dashboard/App.vue

```diff
@@ -87,6 +87,9 @@ export default {
     },
     async initializeAccount() {
       await this.$store.dispatch('accounts/get');
+      this.$store.dispatch('setActiveAccount', {
```

Comment:
@pranavrajs The `setActiveAccount` API call moved to the root component. It will call when there is a change in the account ID